index.bs

```diff
@@ -166,8 +166,23 @@ Example:
   // (For illustration purposes only. There are better ways of implementing
   // object equality in JavaScript.)
   JSON.stringify(Sanitizer.defaultConfig()) == JSON.stringify(new Sanitizer().config());  // true
+
+  // The .sanitize method is the primary API, and returns a DocumentFragment.
+  // The .sanitizeToString method returns a DocumentFragment serialized as a
+  // string.
+  (new Sanitizer().sanitize("hello")) instanceof DocumentFragment;  // true
+  typeof new Sanitizer().sanitize("hello");  // "object"
```

Comment:
These examples show the API, but not how they are supposed to be used
Maybe something more hands-on? I could also envision pseudo-serializing the DocumentFragment in your code comment like so
> let clean = new Sanitizer().sanitize(userdata); //  DocumentFragment [ "hello" ]
> preview.append(clean);

Reply:
Yeah. I've improved the examples a little bit.